src/main/java/nl/goodbytes/xmpp/tools/output/HtmlTableOutputter.java

```diff
@@ -56,7 +58,8 @@ public void add(ExpectedOutcome expectedOutcome, ServerSettings localServerSetti
             matrixLine += "<tr><th>"+localServerSettings.encryptionPolicy+"</th><th>"+localServerSettings.certificateState+"</th><th>"+localServerSettings.dialbackSupported+"</th>";
 
         }
-        matrixLine += "<td>"+expectedOutcome.getConnectionState().getShortCode()+"</td>";
+        entryCount++;
+        matrixLine += "<td>"+entryCount+". "+expectedOutcome.getConnectionState().getShortCode()+"</td>";
```

Comment:
Does the code make some kind of attempt to ensure that it will iterate over the collections in the same manner? It would be helpful if the same number was generated for the same combination of settings each run. Maybe we should explicitly sort both collections prior to processing (if that does not already happen).